dashboard/src/pages/Capability/Components/ShowComponent/index.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect, useState } from 'react';
+
+import { Modal, Space, Table, Tag, Tooltip, Typography } from 'antd';
+
+import { ShowParameters } from './types';
+
+export default ({ name, parameters }: ShowParameters) => {
+  const [localVisible, setLocalVisible] = useState(false);
+
+  const { Paragraph, Text } = Typography;
+
+  useEffect(() => {
+    setLocalVisible(name !== '');
+  }, [name, parameters]);
+
+  return (
+    <Modal
+      forceRender
+      visible={localVisible}
+      onCancel={() => setLocalVisible(false)}
+      footer={null}
+      width={1000}
+      maskClosable
+      destroyOnClose
+    >
+      <h2>{`Type: ${name}`}</h2>
+      <Table
+        columns={[
+          {
+            title: 'Name',
+            dataIndex: 'name',
+            key: 'name',
+            width: 200,
+            render: (text, row) => [
+              <Paragraph copyable={{ text: row.name }}>
+                <Space size="small" align="center">
+                  {row.name}
+                  {!row.required ? undefined : <Tag color="geekblue">request</Tag>}
+                </Space>
+              </Paragraph>,
+            ],
+          },
+          {
+            title: 'Short',
+            dataIndex: 'short',
+            key: 'short',
+            width: 100,
+            responsive: ['md'],
+            render: (text, row) => [
+              <Space size="small" align="center">
+                {!row.short ? undefined : <Tag color="magenta">{text}</Tag>}
+              </Space>,
+            ],
+          },
+          {
+            title: 'Usage',
+            ellipsis: {
+              showTitle: false,
+            },
+            dataIndex: 'usage',
+            key: 'usage',
+            render: (address) => (
+              <Tooltip placement="topLeft" title={address}>
+                {address}
+              </Tooltip>
+            ),
+          },
+          {
+            title: 'Default',
+            dataIndex: 'default',
+            key: 'default',
+            width: 200,
+            responsive: ['md'],
+            render: (text, row) => [
+              <Space size="small" align="center">
+                {!row.default ? undefined : <Text code>{text}</Text>}
```

Comment:
text is empty not return <Text>, return null?

<img width="975" alt="WeChat15bf5f38334deb2722ea3e8bb718a1aa" src="https://user-images.githubusercontent.com/10176784/103116424-6e59f400-46a1-11eb-996b-13eb87be37ce.png">